MovieApp/PresentationLayer/Presenters/HomeScreenPresenter.swift

```diff
@@ -1,2 +1,13 @@
 class HomeScreenPresenter {
+
+    private let movies = MockDataService.movies
+
+    func getNumberOfMovies() -> Int {
```

Comment:
functions that have no argument and return a value should be defined as computed properties